ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -2269,6 +2269,176 @@ keys:
+                            - str
+                            description: 'Set the random-detect ecn weight.
+
+                              Enter a value between 1 and 15
+
+                              '
+        tx_queues:
+          type: list
+          primary_key: id
+          convert_types:
+          - dict
+          items:
+            type: dict
+            keys:
+              id:
+                type: int
+                convert_types:
+                - str
+                required: true
+                description: TX-Queue ID
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    description: Explicit Congestion Notification
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: 'Enable counter for random-detect ECNs
+
+                          '
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            type: str
+                            required: true
+                            valid_values:
+                            - segments
+                            - bytes
+                            - kbytes
+                            - mbytes
+                            description: 'Indicate the units to be used for the threshold
+                              values.
+
+                              This should be one of segments, byte, kbytes, mbytes.
+
+                              '
+                          min:
+                            type: int
+                            min: 1
+                            max: 256000000
+                            convert_types:
+                            - str
+                            description: 'Set the random-detect ecn minimum-threshold
+
+                              '
+                          max:
+                            type: int
+                            required: true
+                            convert_types:
+                            - str
+                            min: 1
+                            max: 256000000
+                            description: 'Set the random-detect ecn maximum-threshold
+
+                              '
+                          max_probability:
+                            type: int
+                            required: true
+                            min: 1
+                            max: 100
+                            convert_types:
+                            - str
+                            description: 'Set the random-detect ecn max-mark-probability.
+
+                              The value should be between 1 and 100.
+
+                              '
+                          weight:
+                            type: int
+                            min: 1
```

Comment:
the min for the weight could be between 0 and 15, see cli:

ld002(config-s-test-qos-profile-NET-INFRA-txq-4)# random-detect ecn minimum-threshold 256 kbytes maximum-threshold 512 kbytes weight ?
  <0-15>  Value of ECN weight